packages/client/src/Components/Mapping/helpers/clearButtons.js

```diff
@@ -0,0 +1,4 @@
+export default function clearButtons (/* array */ buttons) {
+  buttons.forEach(button => button.remove())
+  return []
```

Comment:
If you are returning an empty array anyway, then line 2 and the parameter are not needed. In fact, I was thinking about this the other day with my `getClearedRoute` function. It's such a basic function that just calling it 'clearArray' or something like that would make more sense as that way it's also reusable.

Reply:
We need to call `button.remove()` to make it detach itself from the map UI.

Reply:
ah, makes sense now :)